### ISSUE TITLE
Create Milestone Review Report

### ISSUE THREAD

**Issue by kubraaksux** [bounswe/bounswe2023group1#358]
Milestone review report is necessary for our customers. Report should be a synthesis and a reflection of our work so far.

**Comment by kubraaksux** [comment]
In crafting this report for Milestone 1, I have diligently captured the essence of our project's mission and progress. The project overview clearly articulates our goal of enhancing disaster response efforts while emphasizing reliability and user-friendliness. We've showcased the significant strides made during this milestone, including team organization, tech stack selection, and user role implementation. The meticulous attention given to customer feedback and the subsequent adjustments made to streamline the authentication process reflect our commitment to user-centric development. Our diligent approach to deliverables, project management tools, and processes is evident in the comprehensive status updates and evaluation of our practices. We've recognized the role of GitHub, project planning, and documentation in keeping us on track and are actively working on improving communication with other teams. This report underscores our dedication to this project, making it a professional and well-structured account of our achievements in Milestone 1, setting a strong foundation for the milestones that lie ahead.
